src/routes/events.$eventId.tsx

```
@@ -1328,6 +1335,65 @@ clubs (name, slug, logo_url, primary_color, secondary_color),          event_met
+    if (typeof window === "undefined" || !navigator.geolocation) {
+      toast.error("Geolocation is not supported by your browser.");
+      return;
+    }
+
+    navigator.geolocation.getCurrentPosition(
+      async (position) => {
+        const { latitude, longitude } = position.coords;
+        const venueObj = Array.isArray(event.venues) ? event.venues[0] : event.venues;
+        const targetLat = venueObj?.latitude ?? event.latitude;
+        const targetLng = venueObj?.longitude ?? event.longitude;
+        const targetRadius = venueObj?.geofence_radius_meters ?? event.geofence_radius_meters ?? 100;
```

ESLint

[error] Insert `⏎·········`

(prettier/prettier)